### ISSUE TITLE
Security?

### ISSUE THREAD

**Issue by drozzy** [zeromq/ezmq#31]
Does this support http://curvezmq.org/?

Also, which erlang lib has the most up-to-date binding?

**Comment by RoadRunnr** [comment]
No, ezmq does not support the ZMQ framing that is required for security.
